Makefile

```diff
@@ -20,6 +20,7 @@ ALL_SRC := $(shell find . -name "*.go" | grep -v -e vendor \
 
 .PHONY: test
 test: $(COV_REPORT)
+	$(ECHO_V)go test -race -timeout=1s $(PKGS)
```

Comment:
Also, we don't want to run with `-race` locally by default. It uses a much higher amount of CPU and memory, so we only want to do that on CI, which is why we have the $(RACE) flag